eva/binder/statement_binder_context.py

```diff
@@ -60,6 +80,15 @@ def add_derived_table_alias(self,
     def get_binded_column(self,
                           col_name: str,
                           alias: str = None) -> Tuple[str, CatalogColumnType]:
```

Comment:
@gaurav274 The return columns object is `CatalogColumnType` or `DataFrameColumn`? Below:
- `_check_table_alias_map` and `_check_derived_table_alias_map` returns `DataFrameColumn`
- `_search_all_alias_maps` returns `CatalogColumnType`

Reply:
_check_derived_table_alias_map should have been CatalogColumnType

Reply:
How about `_check_table_alias_map`?  `get_binded_column` directly returns the `col_object` from the  `_check_table_alias_map`. So their type should be same?